client/layout/logged-out.jsx

```diff
@@ -153,6 +156,10 @@ const LayoutLoggedOut = ( {
 
 	let masterbar = null;
 
+	useEffect( () => {
+		refreshColorScheme( 'default', colorScheme );
```

Comment:
Should we check if `isWooPasswordlessJPC` before calling the function? If it's only needed for Woo.

Reply:
@chihsuan FIxed it in [Refresh color scheme when if it is woo passwordless JPC](https://github.com/Automattic/wp-calypso/pull/96557/commits/6caaed408f2a148978b60a8448e25fdfaeda6066)